src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx

```diff
@@ -305,12 +305,12 @@ For more details, see the Logs section of our [best practices](/docs/integration
     </tr>
     <tr>
       <td>Array messages</td>
-      <td>❌</td>
+      <td>✅</td>
```

Comment:
I think this should have the same non-nested / homogeneous warning as attributes 🤔

Reply:
Gah, where are my manners? Thank you! 😄